usage/rest-server/src/main/java/org/apache/brooklyn/rest/transform/CatalogTransformer.java

```diff
@@ -91,7 +91,7 @@
         return new CatalogEntitySummary(item.getSymbolicName(), item.getVersion(), item.getDisplayName(),
             item.getJavaType(), item.getPlanYaml(),
             item.getDescription(), tidyIconLink(b, item, item.getIconUrl()),
-            config, sensors, effectors,
+            item.tags().getTags(), config, sensors, effectors,
```

Comment:
We did consider that, but creating them at the point when the catalog item is generated seemed like an overhead for something that's unlikely to change, and this way the data stored in the catalog will be consistent. Happy to change, but for now I think it's acceptable